pkgs/applications/networking/mailreaders/notmuch/default.nix

```diff
@@ -42,20 +48,28 @@ stdenv.mkDerivation rec {
 
     substituteInPlace lib/Makefile.local \
       --replace '-install_name $(libdir)' "-install_name $out/lib"
-
+  '' + optionalString withEmacs ''
     substituteInPlace emacs/notmuch-emacs-mua \
       --replace 'EMACS:-emacs' 'EMACS:-${emacs}/bin/emacs' \
       --replace 'EMACSCLIENT:-emacsclient' 'EMACSCLIENT:-${emacs}/bin/emacsclient'
   '';
 
-  configureFlags = [ "--zshcompletiondir=${placeholder "out"}/share/zsh/site-functions" ];
+  configureFlags = [
+    "--zshcompletiondir=${placeholder "out"}/share/zsh/site-functions"
+    "--mandir=${placeholder "doc"}"
```

Comment:
Maybe this should go to `man` output. Some people want to read the manpage without reading the developer documentation. Also the `man` output automatically picks up the right files without an need for having to specify its path in `configureFlags`.

Reply:
yeah I noticed right after pushing. Fixing it now

Reply:
There